app/controllers/application_controller.rb

```diff
@@ -519,8 +488,8 @@ def flash_search_warnings(result)
   end
 
   # Don't get unnecessary data for json requests
+
   skip_before_action  :load_admin_banner,
-                      :set_redirects,
```

Comment:
Looking at the test failures, I think we need to remove this in one other place now:

https://github.com/otwcode/otwarchive/blob/820914f3fd79c4f663bb820079740e6a8549ff4b/app/controllers/autocomplete_controller.rb#L6